frontend/src/Dashboard/CrossingDetailPage/CrossingDetailPage.js

```diff
@@ -1,19 +1,76 @@
 import React, { Component } from 'react';
+import { graphql, compose } from 'react-apollo';
+import gql from 'graphql-tag';
 import CrossingDetails from './CrossingDetails';
 import CrossingStatusHistory from './CrossingStatusHistory';
 
+
 class CrossingDetailPage extends Component {
   render() {
-    const crossingId = this.props.match.params.id;
+    if ( !this.props.CrossingByIdQuery ||
+          this.props.CrossingByIdQuery.loading ||
+          !this.props.CrossingHistoryQuery ||
+          this.props.CrossingHistoryQuery.loading ) {
+      return (<div>Loading</div>)
+    }
+
+    const crossing = this.props.CrossingByIdQuery.crossingById;
+    const community = crossing.communityCrossingsByCrossingId.nodes[0].communityByCommunityId;
```

Comment:
I'm super ignorant of graphql nuance -- but referencing .nodes[0] seems really strange on the surface. Can you explain why it's used? This can be an in-person.

Reply:
In the future we'll want to have all of the communities instead of just one. There was a note in the asana task: `Note: Add Community workflow will be a separate story` that I took to mean we should put off thinking about this until we figure out the edit community workflow.

I've updated it to list all of the communities in a non editable way in the edit crossing PR.